### PR TITLE
Apply missing updates from f/flux2 docs

### PR DESCRIPTION
**Pull request by dholbach**
In a review of https://gist.github.com/dholbach/1a3b8da305825e75eee0e0407b04d39f these are the missing pieces which turned up (apart from #123).

### CODE REVIEW

File: content/en/docs/migration/flux-v1-migration.md [hunk 1]
```diff
@@ -181,7 +180,7 @@ Configure the reconciliation of the `deploy` dir on your cluster:
 
 ```sh
 $ flux create kustomization app \
-  --source=GitRepository/app \
+  --source=app \
```
Comment (hiddeco) [inline content/en/docs/migration/flux-v1-migration.md:183]: This does not seem to be right.

Comment (dholbach) [reply content/en/docs/migration/flux-v1-migration.md:183]: https://github.com/fluxcd/flux2/blame/main/docs/guides/flux-v1-migration.md#L170

Comment (hiddeco) [reply content/en/docs/migration/flux-v1-migration.md:183]: It should be `GitRepository/app`

Comment (dholbach) [reply content/en/docs/migration/flux-v1-migration.md]: Thanks for clarifying!

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**